rust/server/src/documents.rs

```diff
@@ -355,6 +355,7 @@ pub async fn serve_path(
                 <link href="https://fonts.googleapis.com/css2?family=Inter:slnt,wght@-10..0,100..900&family=IBM+Plex+Mono:wght@400&display=swap" rel="stylesheet">
                 <link rel="stylesheet" type="text/css" href="/~static/{version}/shoelace-style/themes/light.css">
                 <link rel="stylesheet" type="text/css" href="/~static/{version}/shoelace-style/themes/dark.css">
+                <link rel="stylesheet" type="text/css" href="/~static/{version}/shoelace.css">
```

Comment:
I don think this is necessary because in `web/src/views/dynamic.ts` we have 

```ts
import '../shoelace.css'
```

and so Parcel bundles that CSS into `web/dist/views/dynamic.css`. Adding this here doesn't really hurt too much but does adds another unnecessary request and more importantly leaks the frontend architecture into the server. Id prefer if the server just served up the CSS and JS for the relevant view and the front end is responsible for putting what is necessary in those.

Can you please check if this change is necessary.